### DISCUSSION TITLE
Using MathJax or Latex in Texts (Labels)

### DISCUSSION THREAD

**Discussion by bulme-sm** [hneemann/Digital#679] [Ideas]
It seems that some limited set of MathJax or Latex is supported in the labels. 
E.g. O_{A<B} 

It would be nice if some more MathJax could be used. Like these examples and more: 
$\overline{A} = \neg A$

$\overline A = \neg A$

$A\vee B$

$\overline{A \vee B}$

$A \wedge B$

$\overline{A \wedge B}$

<img width="131" alt="MathJax-Logic-examples" src="https://user-images.githubusercontent.com/58000412/111043768-a244ea00-8444-11eb-9a98-f07d9cded289.png">

Or is there already a way of doing this which I have overlooked?

**Comment by hneemann** [comment]
Overline is done by `~A` and if you want to use vee and wedge you can use the unicode characters ∨ and ∧.
And `~{A∧B}` is also possible.
But i will add a `\vee` and `\wedge` command because it is more convenient. 
You can also take a look at the file `examples/misc/TextFormatExample.dig` or `src/main/dig/misc/TextFormatExample.dig` in the repo.

**Reply by bulme-sm** [reply]
Great, thank you!

By the way: If you enter "~A" on macOS it results to Ã. The operating system automatically creates the A with tilde on top.
However what works is to use ~{A} strangly if I type in the keyboard command for tilde on macOS which is option n and then place a { behind then 2 times the tilde appears in the label of an digital input for example.
So I get ~~{A} automatically and I have to delete one tilde.

**Reply by hneemann** [reply]
The latest [pre-release](https://infdigital.dhbw-mosbach.de/) supports the `\vee` and `\wedge` commands.

What about pressing ~ followed by a blank?

**Reply by bulme-sm** [reply]
Pressing ~ followed by a blank, works! Thanks a lot. 
Digital is a really fantastic tool. I am still a very beginner but it looks like I will use it over the next years in my classes in Austria.
